test/e2e/snaps/test-snap-get-locale.spec.js

```diff
@@ -115,10 +112,12 @@ describe('Test Snap Get Locale', function () {
         // try to select dansk from the list
         await driver.clickElement({ text: 'Dansk', tag: 'option' });
 
+        // there are 2 re-renders which cause flakiness (issue #25651)
+        // the delay can be removed once the issue is fixed in the app level
+        await driver.delay(1000);
+        await driver.assertElementNotPresent('.loading-overlay');
+
         // click on the global action menu
-        await driver.waitForSelector(
-          '[data-testid="account-options-menu-button"]',
-        );
```

Comment:
waitForSelector is not needed as it's implicit on the subsequent `clickElement` method